### ISSUE TITLE
The visualization of Resource Accessed view is not keyboard accessible

### ISSUE THREAD

**Issue by zqian** [tl-its-umich-edu/my-learning-analytics#797]
## Expected behavior (A clear and concise description of what you expected to happen) : 

User should be able to navigate the Resource Accessed view using keyboard Tab key.

## Describe the bug (Tell us what happens instead of the expected behavior) :

Using Tab key, user is not able to get into the Resource Accessed visualization.

## Steps to Reproduce :
1. Open Resource Accessed view for a course 
2. Instead of mouse, use key board Tab key. 
3. One can navigate through the page title bar, week selection, resource title, etc. However, one cannot get into the main bar chart view area using Tab key.

**Comment by chrisrrowland** [comment]
This is not directly related to this issue, but when looking into it I noticed that some resource names have ellipses, so for screen reader purposes, it will literally spit out the word "ellipsis" when reading the resource name.
I'm thinking we would want it to give the real file name (while still visually displaying the name with the ellipsis)?  Or are the filenames not that meaningful (I suppose they better be since it's the only way to identify a resource)

**Comment by gsilver** [comment]
@pushyamig @jennlove-um  - mark as done, but same comments as https://github.com/tl-its-umich-edu/my-learning-analytics/issues/749 - address punctuation, and instead of reproducing in text the entirety of the data, do some research on how users would want to use the information and tailor the alternative text to that. For example, a cutoff point where data ceases to have meaning (resources used by less than 30% of students) and omit the outliers. Also - something like: "You did not access these resources that students with grades higher than 80% accessed [list]. Making this up, research would be needed, like I said!

**Comment by jennlove-um** [comment]
Testing for this passes: I am able to use the keyboard to tab to the main part of the resources accessed chart.  However, I cannot view the hover-over information using the keyboard.  I will open a new issue for that.
